planner/core/logical_plans.go

```diff
@@ -493,7 +495,7 @@ func (ds *DataSource) deriveIndexPathStats(path *accessPath, conds []expression.
 	path.countAfterAccess = float64(ds.statisticTable.Count)
 	path.idxCols, path.idxColLens = expression.IndexInfo2Cols(ds.schema.Columns, path.index)
 	if !path.index.Unique && !path.index.Primary && len(path.index.Columns) == len(path.idxCols) {
-		handleCol := ds.getHandleCol()
+		handleCol := ds.getPKIsHandleCol()
```

Comment:
Why change it?

Reply:
Oh, this two is nearly the same. I'll remove one of them.